app/src/main/java/org/mozilla/tv/firefox/channels/pinnedtile/PinnedTileRepo.kt

```diff
@@ -14,9 +14,12 @@ import io.reactivex.BackpressureStrategy
 import io.reactivex.Observable
 import io.reactivex.subjects.BehaviorSubject
 import org.json.JSONArray
+import org.json.JSONObject
 import org.mozilla.tv.firefox.channels.BundleType
 import org.mozilla.tv.firefox.channels.ChannelRepo
-import java.util.UUID
+import java.util.*
```

Comment:
Nit: I think our policy is to avoid star imports.  You can set Intellij to not use them under 'code style'.

EDIT: looks like it's fixed in a later commit.